gems/moji/1.6/moji.rbs

```diff
@@ -0,0 +1,5 @@
+module Moji
+  def self.han_to_zen: (::String str, ?untyped tp) -> String
```

Comment:
How about rename the second argument to `type`? It is named as `type` in the documentation, and `type` is easier to understand the meaning than `tp`.

https://github.com/gimite/moji/blob/fe5013d404a744f1c521145e17943d17385327f8/lib/moji.rb#L186

Reply:
Certainly the `type` is easier to understand. I'll set the second argument to `type`.

By the way, is there a problem with using `type` as an argument name? I curious because we also use `type` in Type alias declarations.

Reply:
Probably it doesn't have any problems. If I remember correctly, RBS allows keywords as an argument name. You can also quote the keyword with backticks if you get a syntax error.